app/controllers/claims/support/claims/claim_activities_controller.rb

```diff
@@ -0,0 +1,17 @@
+class Claims::Support::Claims::ClaimActivitiesController < Claims::Support::ApplicationController
+  def index
+    @pagy, @claim_activities = pagy(Claims::ClaimActivity.order(created_at: :desc))
+
+    authorize [:claims, @claim_activities]
+  end
+
+  def show
+    @claim_activity = Claims::ClaimActivity.find(params[:id])
+
+    if @claim_activity.sampling_uploaded?
+      @pagy, @provider_samplings = pagy(@claim_activity.record.provider_samplings.joins(:provider).order(providers: { name: :asc }))
```

Comment:
Might be worth moving some of this to a scope

Reply:
Resolved here: [fixup! Show Claim Activities as a timeline](https://github.com/DFE-Digital/itt-mentor-services/pull/1261/commits/0bd1bb81540fc756514e39c6c305b00a8649c4e3)